griottes/graphmaker/graph_generation_func.py

```diff
@@ -297,7 +304,7 @@ def generate_delaunay_graph(
         else:
             G.add_node(cell)
 
-    pos = {int(i): (cells[i]["z"], cells[i]["x"], cells[i]["y"]) for i in cells.keys()}
+    pos = {int(i): (cells[i]["x"], cells[i]["y"], cells[i]["z"]) for i in cells.keys()}
```

Comment:
I also suggest to follow ZYX order, like the np.array

Reply:
Done